### PR TITLE
Emergency alerts

### PR DESCRIPTION
**Pull request by prescottie**
### Wrike: ###

* https://www.wrike.com/open.htm?id=499917037

### Description: ###

Change & simplify guest Alerts to emergency alerts for locations. We will still keep this behind a feature flag until we can do the PIA and sort out some of the twilio stuff, but I want to finish off the code for this, so we can turn it on when we have the rest of our shit together :) 

### PR Checklist ###

- [x] Assigned 2 or more reviewers
- [x] Added yourself as "Assignee"
- [x] Included labels such as "WIP", "Waiting for reviews", etc.

### CODE REVIEW

**Comment by stevenallen05** [comment]
After discussions with compliance, I think we're going to have to put this on hold for the moment. They've got enough concerns to warrant taking another look at how we can protect ourselves.

**Comment by brandonjmckay** [comment]
@prescottie I added some linting improvements for the editor, this will need you to change the path vars to be underscore.
You can check for any other issues in the `ci/circleci: build` step details.
I've got the linter to work, but it's not showing the extra linting we have in the web studio yet... working on that.

**Comment by tractionguest-integrations** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  You don't seem to have any wrike tickets for this PR!
    You need to add your wrike ticket to one of your commits, e.g., `[#1234] Your commit message`
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  Remember to use the `Squash and Merge` button to merge!
    Always remember, your final commit message needs to start with a wrike ticket like, `[#1234] Your commit message`.
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>

  <a href="https://stoplight.io/p/docs/gh/tractionguest/guest-api/openapi.yml?srn=gh/tractionguest/guest-api/openapi.yml&group=emergency_alerts" target=_blank>View docs for this page</a>
  </td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against d69a46ea400da73e50fd63fbb5ee121b3df3715d
</p>

File: models/EmergencyAlertDetail.v1.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+title: GuestAlertDetail
```
Comment (brandonjmckay) [inline models/EmergencyAlertDetail.v1.yaml:1]: this doesn't match the file

File: openapi.yml [hunk 2]
```diff
@@ -3095,36 +3095,46 @@ paths:
...
                   message: some text
                   channels:
                     - SMS
                     - EMAIL
-                  signin_selectors:
-                    signin_ids:
-                      - 19
-                      - 64
-                    is_signed_out: false
+          application/javascript:
```
Comment (brandonjmckay) [inline openapi.yml:3113]: why javascript? this should be json

File: openapi.yml [hunk 3]
```diff
@@ -3095,36 +3095,46 @@ paths:
...
+            schema:
+              type: object
+              properties: {}
+          application/xml:
```
Comment (brandonjmckay) [inline openapi.yml:3117]: what is xml doing in here? we don't support xml

**Comment by stevenallen05** [comment]
Closing this for now since we don't have the compliance issues sorted out yet

**Pull request closed without merging**